src/manual/tasks/workertypes.md

```diff
@@ -0,0 +1,17 @@
+---
+filename: tasks/workertypes.md
+title: Worker Types
+order: 20
+---
+
+Workers execute tasks, but there are many types of workers. A task specifies a
+single "worker type" by which it should be executed, using the two-part
+namespace `<provisionerId>/<workerType>`.  Workers with the same worker type
+all consume tasks from a single queue, as [described
+later](/manual/task-execution/queues), and as such are interchangeable and have
+identical configurations.
+
+The format of a task's `payload` property is specific to the worker that will
+execute it, so defining a task requires knowledge of worker type's
+configuration. If given a task with an inappropriate payload, a worker will
+resolve the task with the reason `malformed-payload`.
```

Comment:
I think we need to [do something about that](https://github.com/taskcluster/taskcluster-rfcs/issues/22), but until such time as we complete that project, I think that this vague sentence in the manual approximates the vague reality 😕